uk_election_timetables/election_ids.py

```diff
@@ -97,6 +97,10 @@ def valid_election_type(el_type):
     def requires_country(el_type):
         return el_type in ["local"]
 
+    if election_id.startswith("mayor.london"):
+        # Mayor of London uses the GLA timetable
+        return GreaterLondonAssemblyElection(poll_date)
+
```

Comment:
You could argue that `mayor.date` (without an org) should `raise AmbiguousElectionIdError` now